recipe/run_test.bat

```diff
@@ -0,0 +1,12 @@
+:: setup for signing during tests
```

Comment:
Careful with https://github.com/conda/conda-build/issues/4427

The tests in `meta.yaml` might be ignored on Windows.

Reply:
yeah, was worrying this could happen... will rename the script and call it as test command